app/controllers/admin/settings_controller.rb

```diff
@@ -11,12 +10,23 @@ def index
   def update
     @admin_setting = AdminSetting.first || AdminSetting.create(:last_updated_by => Admin.first)
 
-    if @admin_setting.update_attributes(params[:admin_setting])
+    if @admin_setting.update_attributes(admin_setting_params)
       flash[:notice] = ts("Archive settings were successfully updated.")
       redirect_to admin_settings_path
     else
       render :action => "index"
     end
   end
 
+  private
+  def admin_setting_params
+    params.require(:admin_setting).permit(
+      :account_creation_enabled, :invite_from_queue_enabled, :invite_from_queue_number,
+      :invite_from_queue_frequency, :days_to_purge_unactivated, :last_updated_by,
+      :invite_from_queue_at, :suspend_filter_counts, :suspend_filter_counts_at,
+      :enable_test_caching, :cache_expiration, :tag_wrangling_off, :guest_downloading_off,
+      :default_skin_id, :stats_updated_at, :disable_filtering, :request_invite_enabled,
```

Comment:
default_skin_id is set via the skin object ( and you can see it is not in the form that matches this controller ).

stats_updated_at show when a process was last run and should not be settable from this controller.